Content.Client/Storage/ClientStorageComponent.cs

```diff
@@ -178,7 +168,7 @@ private void ChangeStorageVisualization(SharedBagState state)
         /// Function for clicking one of the stored entity buttons in the UI, tells server to remove that entity
         /// </summary>
         /// <param name="entityUid"></param>
-        private void Interact(BaseButton.ButtonEventArgs buttonEventArgs, EntityUid entityUid)
+        private void OnTake(BaseButton.ButtonEventArgs buttonEventArgs, EntityUid entityUid)
```

Comment:
This should actually be OnInteract or similar since this also handles right click menu of items inside the storage.

Reply:
Good point. I've changed it to `OnInteract`. Also, what's the status of space-wizards/RobustToolbox/pull/1978? Is it just waiting for reviews/testing?